### PR TITLE
Home page with tabs changing

### PR DESCRIPTION
**Pull request by sktbrd**
I got a new Homebase page called Home, it has 4 default tabs , I cant switch the content of each tab 

Its a work in progress, please dont merge

### CODE REVIEW

**Comment by sktbrd** [comment]
tabs are changing config , uhuuul ! Just waiting on the right config we wanna now

File: src/constants/initialHomebaseTabsConfig.tsx [hunk 1]
```diff
@@ -0,0 +1,1139 @@
...
+          url: "https://irb0gie.vercel.app",
+        },
+      },
+      fidgetType: "iframe",
+      id: "iframe:d65e4ea9-d6ac-48d0-8eaf-d4c2c4187f7a",
+    },
+  },
+  isEditable: true,
+  fidgetTrayContents: [],
+};
```
Comment (willyogo) [inline src/constants/initialHomebaseTabsConfig.tsx:237]: These are contents of a draft/test tab, we can remove

File: src/constants/initialHomebaseTabsConfig.tsx [hunk 2]
```diff
@@ -0,0 +1,1139 @@
...
+  fidgetInstanceDatums: {
+    "text:a1acd942-be81-4ab6-ae4e-2b6e41ccfde2": {
+      config: {
+        data: {},
+        editable: true,
+        settings: {
+          background: "rgba(70, 69, 69, 0.27)",
+          css: "",
+          fidgetBorderColor: "rgb(159, 111, 206)",
+          fidgetBorderWidth: "var(--user-theme-fidget-border-width)",
```
Comment (willyogo) [inline src/constants/initialHomebaseTabsConfig.tsx]: 4px

File: src/constants/initialHomebaseTabsConfig.tsx [hunk 3]
```diff
@@ -0,0 +1,1139 @@
...
+        data: {},
+        editable: true,
+        settings: {
+          title: "Fidget Info",
+          DescriptionColor: "rgb(255, 251, 251)",
+          HeaderColor: "rgb(255, 255, 255)",
+          background: "rgba(65, 64, 64, 0.15)",
+          css: "",
+          fidgetBorderColor: "rgb(62, 153, 245)",
+          fidgetBorderWidth: "var(--user-theme-fidget-border-width)",
```
Comment (willyogo) [inline src/constants/initialHomebaseTabsConfig.tsx]: 4px

File: src/constants/initialHomebaseTabsConfig.tsx [hunk 4]
```diff
@@ -0,0 +1,1139 @@
...
+          css: "",
+          fidgetBorderColor: "rgb(159, 111, 206)",
+          fidgetBorderWidth: "var(--user-theme-fidget-border-width)",
+          fidgetShadow: "0 4px 8px rgba(0,0,0,0.25)",
+          fontColor: "rgb(221, 221, 221)",
+          fontFamily: "'__Inter_36bd41', '__Inter_Fallback_36bd41'",
+          headingsFontColor: "rgb(228, 228, 228)",
+          headingsFontFamily:
+            "'__Trispace_3014ec', '__Trispace_Fallback_3014ec'",
+          text: "Did you ever wish you could do more on social than post, like, and comment? Fidgets make this possible. In fact, Fidgets make anything possible. Arrange and configure a growing library of Fidgets on your social profile (your Space) or your homebase to seamlessly bridge the vast digital universe into social, and social into to digital universe. \n\nLearn more about each Fidget from the Fidget on the right 👉\n\nJoin the Discord to get early access to the  Fidget SDK\n\nCheck out a few examples of Fidgets below 📺  ⏹️  🌐 🛰️ 🏛️",
```
Comment (willyogo) [inline src/constants/initialHomebaseTabsConfig.tsx]: add link to 'Join the Discord'

https://discord.gg/eYQeXU2WuH

File: src/constants/initialHomebaseTabsConfig.tsx [hunk 5]
```diff
@@ -0,0 +1,1139 @@
...
+    },
+    "governance:6f7ddc86-57eb-4b07-951b-d65b8916270a": {
+      config: {
+        data: {},
+        editable: true,
+        settings: {
+          background: "var(--user-theme-fidget-background)",
+          fidgetBorderColor: "rgba(255, 109, 109, 1)",
+          fidgetBorderWidth: "2px",
+          fidgetShadow: "var(--user-theme-fidget-shadow)",
```
Comment (willyogo) [inline src/constants/initialHomebaseTabsConfig.tsx]: 0 2px 5px rgba(0,0,0,0.15)

File: src/constants/initialHomebaseTabsConfig.tsx [hunk 6]
```diff
@@ -0,0 +1,1139 @@
...
+          fidgetBorderWidth: "2px",
+          fidgetShadow: "0 2px 5px rgba(0,0,0,0.15)",
+          size: 0.7,
+          url: "https://fomonouns.wtf",
+        },
+      },
+      fidgetType: "iframe",
+      id: "iframe:ede8bb6b-7f84-41fe-97ce-51aa96711f15",
+    },
+    "text:f9a58fb6-18b7-43fc-bca8-55fdce50343e": {
```
Comment (willyogo) [inline src/constants/initialHomebaseTabsConfig.tsx]: can delete this one I think

**Review by willyogo — CHANGES_REQUESTED** [review]

File: src/constants/initialHomebaseTabsConfig.tsx (hunk 1 above)
Comment (sktbrd) [reply src/constants/initialHomebaseTabsConfig.tsx:237]: If I remove I get a type error 

```bash 
[{
	"resource": "/Users/web3warrior/Desktop/code/nounspace.ts/src/constants/initialHomebaseTabsConfig.tsx",
	"owner": "typescript",
	"code": "2739",
	"severity": 8,
	"message": "Type '{ layoutID: string; layoutDetails: { layoutConfig: { layout: { h: number; i: string; maxH: number; maxW: number; minH: number; minW: number; moved: boolean; resizeHandles: string[]; static: boolean; w: number; x: number; y: number; }[]; }; layoutFidget: string; }; theme: { ...; }; fidgetInstanceDatums: { ...; }; }' is missing the following properties from type 'SpaceConfig': isEditable, fidgetTrayContents",
	"source": "ts",
	"startLineNumber": 7,
	"startColumn": 14,
	"endLineNumber": 7,
	"endColumn": 34
}]

```

`is missing the following properties from type 'SpaceConfig': isEditable, fidgetTrayContents",`

File: src/constants/initialHomebaseTabsConfig.tsx (hunk 3 above)
Comment (sktbrd) [reply src/constants/initialHomebaseTabsConfig.tsx]: made 2px

File: src/constants/initialHomebaseTabsConfig.tsx (hunk 2 above)
Comment (sktbrd) [reply src/constants/initialHomebaseTabsConfig.tsx]: made 2px

File: src/constants/initialHomebaseTabsConfig.tsx (hunk 6 above)
Comment (sktbrd) [reply src/constants/initialHomebaseTabsConfig.tsx]: done

**Pull request merged**